tracer/src/Datadog.Trace/Propagators/B3MultipleHeaderContextPropagator.cs

```diff
@@ -67,7 +67,7 @@ private B3MultipleHeaderContextPropagator()
 
             TelemetryFactory.Metrics.RecordCountContextHeaderStyleExtracted(MetricTags.ContextHeaderStyle.B3Multi);
             var samplingPriority = ParseUtility.ParseInt32(carrier, carrierGetter, Sampled);
-            spanContext = new SpanContext(traceId, parentId, samplingPriority, serviceName: null, null, rawTraceId, rawSpanId);
+            spanContext = new SpanContext(traceId, parentId, samplingPriority, serviceName: null, null, rawTraceId, rawSpanId) { IsRemote = true };
```

Comment:
I think we need to rebase this branc on `steven/is-remote` so these changes don't show up in this PR.

Reply:
We just did that! Thanks!